postapi/Controllers/WeatherForecastController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.AspNetCore.Mvc;
```

Comment:
/WeatherForecastController.cs
Can you remove this file? No need for this controller file, as we need to create empty structure with PostController and CommentController having no actions methods on those.
same comment as @akshayverma91 commented on Praveen's PR.

Reply:
Updated the Changes